### PR TITLE
Test Schemas using eq-schema-validator

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
The schema in `eq-schema-validator` was becoming outdated as it was not being used.
In this PR i have: 
  - removed the schema from this repo
  - added a test script to test all the schemas against the validator
  - fixed all the existing schemas to match the schema

### How to review 
Do all of the schemas still work?
Does Travis run the tests on the schemas?

This PR depends on https://github.com/ONSdigital/eq-schema-validator/pull/5

### CODE REVIEW

**Comment by shahi645** [comment]
Got a lot of schema validation fails 👎

**Comment by jonnyshaw89** [comment]
Good. Shows it works 👍

**Review by shahi645 — APPROVED** [review]
LGTM

File: scripts/test_schemas.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+docker pull onsdigital/eq-schema-validator:eq-1418-improve-json-schema-validation
```
Comment (jonnyshaw89) [inline scripts/test_schemas.sh]: This tag needs to be removed before merge

**Review by yoganandkunche — APPROVED** [review]

**Comment by yoganandkunche** [comment]
@jonnyshaw89 LGTM :+1: Squishy-squashy?

**Pull request merged**